torch/distributed/distributed_c10d.py

```diff
@@ -3927,11 +3913,19 @@ def _new_group_with_tag(
     global_rank = default_pg.rank()
     global_world_size = default_pg.size()
 
+
     # Default to the same backend as the global process group
     # if the backend is not specified.
     if not backend:
         backend = default_backend
     backend = Backend(backend)
+
+    # this timeout defaulting/validation is used for all the new_groups/new_subgroups variants,
```

Comment:
I don't think we can reduce the appearance of this defaulting/validation which is sad... Any idea?